### PR TITLE
Add Arrow to dropdown list

### LINKED ISSUE #9065: Arrow missing from add to list dropdown on author pages
### Problem

It probably should have a down arrow like we have on the want to read button for books?
image

#### Evidence / Screenshot
image

#### Relevant URL(s)
https://openlibrary.org/authors/OL12180252A/Adriana_Churampi_Ram%C3%ADrez

### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:

### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod

### PR DESCRIPTION
**Pull request by Spedi**
Closes #9065 

This PR makes sure to correctly show the arrow on the _Add to List_ dropdown button on the Author page.

### Testing
1) Check an Author page and see if the arrow is shown correctly in the _Add to List_ button.

### Screenshot
Screenshot from 2024-04-22 21-55-00

### Stakeholders
@RayBB

### CODE REVIEW

**Comment by RayBB** [comment]
@Spedi your changes seem to have some unintended effects on the dropdown. Making a big indent and showing new text.
Can you fix this?
| Before | After |
|:-------:|:-----:|
| image | image |

**Comment by Spedi** [comment]
That's weird, I'll have a look at the codebase to find some answers. Do you have any idea about what this can be related to?

**Comment by Spedi** [comment]
I suppose the error I made is that the changes "copy" the dropdown list that's on the _Want to Read_ button, so they inherit the indentation and the unwanted text. Do you think that's a possible explanation of the bug?

**Comment by Spedi** [comment]
The problem seems to go away if I add `old-style-lists` to `<div class="generic-dropper__dropdown old-style-lists">
      $:dropdown_content
    </div>`.
The thing is that the same type of template goes to the _Want to Read_ dropdown too, which is something that we don't want. Should we manage the two dropdowns differently? If yes, how can I try to do that, do you have any suggestions?

**Comment by Spedi** [comment]
Screenshot from 2024-04-24 00-11-34

Changing everything as it was before the PR, and changing the visibility of arrow to visible in the `old-style-lists` css styling should work, since I tried to do it manually with the Inspect function of the browser.
However if I test the code I don't see the changes, since the visibility is set to _hidden_. 
How is it possible if I set it to _visible_ in the code?

**Comment by RayBB** [comment]
@Spedi in reviewing your PR, I also noticed that it is weird that the visibility is set to hidden in some cases. I don't have the time to dig into it now but perhaps if you can uncover that (it could be in the JS?) then you might be able to find a nice solution.

**Comment by Spedi** [comment]
I'm still trying to work on it.
I've tried disabling JS on the Dev Tools but it had no effect on the result, so maybe the problem isn't there.
Could it be a problem of CSS specificity?

**Comment by RayBB** [comment]
It can be the CSS specifically. It likely has to do with nested css which can make it a bit tricky to untangle.
I imagine this is why the code was left a little messy like this with the old styles class.
Have tried looking at other dropdowns on the site to compare the html/css?

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @Spedi, and sorry for the very late review!

I think that we should avoid changes to `openlibrary/templates/lib/dropper.html`, which is meant to be a generic base component for all droppers.

I think if you remove this [CSS rule](https://github.com/internetarchive/openlibrary/blob/ca46cbfe87af6e81dabc966baa5b75c827097879/static/css/components/mybooks-dropper.less#L30-L32), the arrow will be rendered.

**Comment by jimchamp** [comment]
> However if I test the code I don't see the changes, since the visibility is set to hidden.
How is it possible if I set it to visible in the code?

@Spedi, when a `.less` file is changed, it has to be compiled into CSS.  Try running the following after you make a change, and see if it helps:
`docker compose exec web make css`

If you're doing a lot of CSS work, it may be better to run the following:
`docker compose exec web npm run watch-css`

`watch-css` will build the CSS assets every time a `.less` file is saved.

**Comment by Spedi** [comment]
@jimchamp Should I run `docker compose exec web npm run watch-css` while running `docker compose up`?

**Comment by jimchamp** [comment]
Yes.  Your web container needs to be running, so run `docker compose up` first.  Alternatively, you can run `docker compose run --rm home make css` without first running `docker compose up`.

**Comment by Spedi** [comment]
@jimchamp It worked! 
Made the correct changes, gonna do a PR soon, thank you and @RayBB for the patience!

**Review by Spedi — COMMENTED** [review]
This should close the issue, linking some pictures of the new dropdown list:
Screenshot from 2024-05-09 15-41-34
Screenshot from 2024-05-09 15-41-42

File: openlibrary/templates/my_books/dropper.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 $ additional_classes = 'my-books-dropper'
 $if not work_key:
-  $ additional_classes += ' old-style-lists'
+  $ additional_classes += '  old-style-lists'
```
Comment (jimchamp) [inline openlibrary/templates/my_books/dropper.html:22]:
```suggestion
  $ additional_classes += ' old-style-lists'
```

**Review by jimchamp — APPROVED** [review]
This looks good to me.  Will revert the `my_books/dropper`, then merge when the tests pass.

**Pull request merged**